packages/flutter_app/pubspec.yaml

```
@@ -32,7 +32,7 @@ dependencies:
   gap: ^2.0.0
   go_router: ^6.0.2
   google_sign_in: ^5.4.2
-  hooks_riverpod: ^2.3.0
+  hooks_riverpod: ^2.3.5
```

昨日更新されたようです🚀
https://pub.dev/packages/hooks_riverpod/changelog

```suggestion
  hooks_riverpod: ^2.3.6
```